wallet/src/de/schildbach/wallet/ui/OnboardingActivity.kt

```diff
@@ -155,4 +157,19 @@ class OnboardingActivity : RestoreFromFileActivity() {
         }
         return result
     }
+
+    private fun showResetWalletDialog(context: Context) {
+        val dialogBuilder = AlertDialog.Builder(context)
+        dialogBuilder.setTitle(R.string.wallet_lock_reset_wallet_title)
+        dialogBuilder.setMessage(R.string.wallet_lock_reset_wallet_message)
+        //Inverting dialog answers to prevent accidental wallet reset
+        dialogBuilder.setNegativeButton(R.string.wallet_lock_reset_wallet_title) { _, _ ->
+            WalletApplication.getInstance().wipe(context)
+        }
+        dialogBuilder.setPositiveButton(android.R.string.no, null)
+        dialogBuilder.setCancelable(false)
+        val dialog = dialogBuilder.create()
+        dialog.setCanceledOnTouchOutside(false)
+        dialog.show()
+    }
```

Comment:
We do need a showResetWalletDialog method of some sort for the Security / Reset Wallet menu item #264 (it will go in the hide balance PR @denlb) - `PinRetryController.showResetWalletDialog(this, false)`

Can you make this `static` or otherwise accessible to an outside class?  or suggest a different way to reuse this code.

Reply:
Fixed by converting ResetWallet dialog from OnboardingActivity into a separate DialogFragment, it can be now used everywhere simple by:
ResetWalletDialog.newInstance().show(getSupportFragmentManager(), "reset_wallet_dialog");